lib/ansible/module_utils/facts.py

```diff
@@ -99,6 +99,7 @@ class Facts(object):
                     ('/etc/os-release', 'SuSE'),
                     ('/etc/gentoo-release', 'Gentoo'),
                     ('/etc/os-release', 'Debian'),
+                    ('/etc/debian_version', 'Debian'),
```

Comment:
There doesn't seem to be any additional parsing happening for this file.  If it's not the same format as os-release then we'd need additional code below to parse it.  (Or perhaps your new code doesn't rely on debian_version so we don't need to add it here.)